src/pages/404.js

```diff
@@ -27,7 +28,9 @@ const NotFoundPage = () => {
 
             <p>Nothing to see here, go back to <a href={HOME.to}>POW!</a></p>
             <StaticImage
-              src="https://images.unsplash.com/photo-1554579653-17cdf3bfbca6?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MzkxfHxsZWdvJTIwYmF0bWFufGVufDB8fDB8fA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=400&q=60"
+              src="../images/Photo_by_Teo_Zac_unsplash_com_at_teo_2.jpeg"
+              alt="Lost little LEGO Batman"
```

Comment:
**praise:** Love this alt text

Reply:
I have written down and will do this in another pull request
«suggestion: Gatsby handles image resizing, so no need to keep a lower resolution image in the src.

thought (non-blocking): We often call this folder assets as it might contain more than images moving forward, but images is also totally fine in this use case.»